bee-message/tests/byte_cost.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message::{
+    byte_cost::{ByteCost, ByteCostConfig},
+    milestone::MilestoneIndex,
+    output::{Output, OutputId},
+    MessageId,
+};
+use bee_test::rand::output::{rand_alias_output, rand_basic_output, rand_foundry_output, rand_nft_output};
+
+use core::mem::size_of;
+
+const CONFIG: ByteCostConfig = ByteCostConfig {
+    v_byte_cost: 500,
+    v_byte_factor_data: 10,
+    v_byte_factor_key: 1,
```

Comment:
Should this instead be:
```suggestion
    v_byte_factor_data: 1,
    v_byte_factor_key: 10,
```
To remain consistent with the TIP? Unless there's a reason that I'm missing 😅